robocopkaka/bootcamp-29-project
# #158288789 Fix image upload bug

#### What does this PR do?
allow users update centers if the image hasn't changed
#### Description of Task to be completed?
* add an instance method that dispatches the action to update centers
* call the method in an `if-else` block with different arguments
* modify tests for the `EditCenter` component
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependencies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* a modal with the form used for adding centers should come up
* enter the details of the center
* click on `Add Center`
* the modal should close and you should see the new center on the page
* click on the edit icon on the center
* a modal should open with the center's details pre-filled in
* edit the name
* click on Update Center
* the modal should close and you should see the new name reflected on the center's card once it's successful
#### Any background context you want to provide?
users always had to select an image when updating a center even if there wasn't any need to
#### What are the relevant pivotal tracker stories? (if applicable)
#158288789
